### PR TITLE
Conventions

### PR DESCRIPTION
**Pull request by gimre-xymcity**

### CODE REVIEW

File: whitepaper/main.tex [hunk 1]
```diff
@@ -27,6 +27,11 @@
 	}
 	\end{spacing}
 
+	{
+		\pagestyle{empty}
+		\include{typography}
```
Comment (Jaguar0625) [inline whitepaper/main.tex:32]: i'm unsure if typography or preface should come first ?

File: whitepaper/typography.tex [hunk 2]
```diff
@@ -0,0 +1,28 @@
+\section*{Typographical conventions}
```
Comment (Jaguar0625) [inline whitepaper/typography.tex]: *C*onventions

File: whitepaper/typography.tex [hunk 3]
```diff
@@ -0,0 +1,28 @@
+\section*{Typographical conventions}
+%\addcontentsline{toc}{section}{Typographical conventions}
```
Comment (Jaguar0625) [inline whitepaper/typography.tex]: delete ?

File: whitepaper/typography.tex [hunk 4]
```diff
@@ -0,0 +1,28 @@
+\section*{Typographical conventions}
+%\addcontentsline{toc}{section}{Typographical conventions}
+
+This documentation uses the following typographical conventions:
```
Comment (Jaguar0625) [inline whitepaper/typography.tex]: documentation => document ?

File: whitepaper/typography.tex [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+\renewcommand{\arraystretch}{1.3}
+
+\begin{tabular}{l p{1.5cm} p{7cm}} 
+	Sample & Attributes & Description \\
+	\hline
+	\codenamespace & bold, colored & when referring to project name \\
+	\hline
+	\class{ProcessBootstrapper} & 
+	\multirow[t]{3}{=}{fixed-width font} &
+	\multirow[t]{3}{=}{classes, fiele paths, fields (of transactions, blocks or other entities)} \\
```
Comment (Jaguar0625) [inline whitepaper/typography.tex]: file

File: whitepaper/typography.tex [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+	\field{FeeMultiplier} & & \\
+	\filepath{commit\_step.dat} & & \\
+	\hline
+	\nemsetting{network}{maxDifficultyBlocks} & italic & configuration settings, prefixed with configuration filename in text mode\\
+	\hline
+	$\nemsetting{network}{maxDifficultyBlocks}$ & regular & configuration settings, in equations and formulas \\
```
Comment (Jaguar0625) [inline whitepaper/typography.tex]: ... without configuration filename ... ?

**Review by Jaguar0625 — CHANGES_REQUESTED** [review]

File: whitepaper/typography.tex (hunk 3 above)
Comment (gimre-xymcity) [reply whitepaper/typography.tex]: wasn't sure if it should have entry in TOC or not, but I think not

File: whitepaper/main.tex (hunk 1 above)
Comment (gimre-xymcity) [reply whitepaper/main.tex:32]: imo typography, but not sure either

**Review by Jaguar0625 — APPROVED** [review]

**Review by BloodyRookie — APPROVED** [review]

**Comment by Jaguar0625** [comment]
Merged as 08c6787.

**Pull request closed without merging**